### ISSUE TITLE
Error when executing j.tools.atom.installAll()

### ISSUE THREAD

**Issue by yveskerwyn** [Jumpscale/jumpscale_core8#518]
Installed Atom and JS on a Ubuntu Desktop

When executing  j.tools.atom.installAll():

...

---> 46         self.installConfig()
     47 
     48     def installPackagesAll(self):

/opt/jumpscale8//lib/JumpScale/tools/atom/AtomEditor.py in installConfig(self)
    119         print("install atom config")
    120         merged = {}
--> 121         current_config = j.sal.fs.fileGetContents(os.path.expanduser("~/.atom/config.cson"))
    122         merged = cson.loads(current_config)
    123         new_config_path = os.path.join(os.path.dirname(inspect.getfile(self.__class__)), "config.cson")

/opt/jumpscale8//lib/JumpScale/sal/fs/SystemFS.py in fileGetContents(self, filename)
   1398         self.logger.debug('Opened file %s for reading' % filename)
   1399         self.logger.debug('Reading file %s' % filename)
-> 1400         with open(filename) as fp:
   1401             data = fp.read()
   1402         self.logger.debug('File %s is closed after reading' % filename)

FileNotFoundError: [Errno 2] No such file or directory: '/root/.atom/config.cson'

**Comment by xmonader** [comment]
Is atom installed under root user? I think maybe you started it as root?

**Comment by yveskerwyn** [comment]
You are right, let me try again

**Comment by yveskerwyn** [comment]
OK now
